app/models/searchApi/AgentAuthorisation.scala

```diff
@@ -31,9 +31,7 @@ case class AgentAuthorisation(
       address: String,
       localAuthorityRef: String,
       client: Client,
-      representationStatus: String,
-      checkPermission: String,
-      challengePermission: String
+      representationStatus: String
```

Comment:
can representationStatus be anything but Approved?

Reply:
not going forward.

Reply:
sooo... there was talk of removing representation status as part of this ticket, but then this ticket kept giving and I decided maybe we do that later, so as not to do a big bang approach. I think it will only ever be APPROVED "coming back" but when querying for it, you could still ask for PENDING... though should not be allowed to by the FRONTEND